src/main/java/duke/user/User.java

```diff
@@ -0,0 +1,15 @@
+package duke.user;
+import java.util.Scanner;
+
+public class User {
```

Comment:
Did you have a purpose for this class?

Reply:
Yeah, this class is for the future possible design of multi-user cases.

Reply:
Oooh coding with foresight, that's great!